docs/installing-nightly.md

```diff
@@ -2,6 +2,8 @@
 
 >**Note**: only install the nightly if you'd like to try the bleeding edge capabilities of bicep. These are much more likely to have undiscovered bugs or other issues. If you find anything, please open an issue.
 
+>**Note**: The Bicep VS code extension versions older than 0.2 must be uninstalled before or after the installation of the new version.
+
 We are not currently publishing "nightly" releases, but you can grab the latest bits by viewing the latest Action workflows for the `main` branch (or any other branch).
```

Comment:
This isn't part of your PR, but can we update this line to include a reference to the "installing nightly builds" doc?

Reply:
I thought we had it here, but I guess it's in a different doc.

Reply:
I'll add it.

Reply:
Actually on second thought, this _is_ the installing nightly doc 😊